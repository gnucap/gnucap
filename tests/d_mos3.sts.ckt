nmos n gate, sub-threshold
Vgs   1  0  0.236734
M1   2  1  0  4  cmosn  l= 9.u  w= 9.u  nrd= 1.  nrs= 1. 
Vds   3  0  5.
Rds   2  3  100.K
Vbs   4  0 -1.234875
.model cmosn  nmos ( level=3  vto= 0.844345  kp= 41.5964u  gamma= 0.863074 
+ phi= 0.6  rd= 0.  rs= 0.  cbd= 0.  cbs= 0.  is= 0.  pb= 0.7  cgso= 218.971p 
+ cgdo= 218.971p  cgbo= 0.  rsh= 0.  cj= 384.4u  mj= 0.4884  cjsw= 527.2p 
+ mjsw= 0.3002  js= 0.  tox= 41.8n  nsub= 15.3142E+15  nss= 1.E+12 
+ nfs= 3.5934E+12  tpg=1  xj= 400.n  ld= 265.073n  uo= 503.521 
+ kf= 0.  af= 1. 
+ fc= 0.5  delta= 0.36745  theta= 0.  eta= 0.  kappa= 0.2 )
*+(* vfb=-0.4241892 )
*    lambda= 0.00906241  vmax= 55.9035K
.print op v(nodes) iter(0)
.op
.print op i(v*) ps(v*)
.op
.print op id(m1) vgs(m1) vds(m1) vbs(m1) vth(m1) vdsat(m1)
.op
.print op gm(m1) gds(m1) gmb(m1) cbd(m1) cbs(m1)
.op
.print op cgsovl(m1) cgdovl(m1) cgbovl(m1) cgate(m1) region(m1)
.op
.print op cgs(m1) cgd(m1) cgb(m1) vgst(m1) von(m1)
.op
.print op cgst(m1) cgdt(m1) cgbt(m1) is(m1) ig(m1) ib(m1)
.op
.print op p(m1) pd(m1) ps(m1) ids(m1) idstray(m1) iderror(m1)
.op
.print op vdm(m1) vgm(m1) vbm(m1) vsm(m1)
.op
.print op vd(m1) vg(m1) vb(m1) vs(m1)
.op
*>.status notime
.end
