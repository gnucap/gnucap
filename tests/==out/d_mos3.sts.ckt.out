nmos n gate, sub-threshold 
cmosn: rsh - rs - rd conflict: using rsh
cmosn: is - js conflict
#           v(1)       v(2)       v(3)       v(4)       iter(0)   
 27.        0.23673    4.9998     5.        -1.2349     5.        
#           i(Vgs)     i(Vds)     i(Vbs)     ps(Vgs)    ps(Vds)    ps(Vbs)   
 27.        0.         0.         0.         0.         0.         0.        
#           id(M1)     vgs(M1)    vds(M1)    vbs(M1)    vth(M1)    vdsat(M1) 
 27.        2.0729n    0.23673    4.9998    -1.2349     1.5384     0.18427   
#           gm(M1)     gds(M1)    gmb(M1)    cbd(M1)    cbs(M1)   
 27.        9.6704n    7.4992p    3.077n     0.         0.        
#           cgsovl(M1) cgdovl(M1) cgbovl(M1) cgate(M1)  region(M1)
 27.        1.9707f    1.9707f    0.         62.973f    5.        
#           cgs(M1)    cgd(M1)    cgb(M1)    vgst(M1)   von(M1)   
 27.        0.         0.         62.973f   -1.3016     1.5384    
#           cgst(M1)   cgdt(M1)   cgbt(M1)   is(M1)     ig(M1)     ib(M1)    
 27.        1.9707f    1.9707f    62.973f   -2.0729n    0.         0.        
#           p(M1)      pd(M1)     ps(M1)     ids(M1)    idstray(M1) iderror(M1)
 27.        10.364n    10.364n    0.         2.0729n    0.         ??        
#           vdm(M1)    vgm(M1)    vbm(M1)    vsm(M1)   
 27.        2.4999    -2.2632    -3.7348    -2.4999    
#           vd(M1)     vg(M1)     vb(M1)     vs(M1)    
 27.        4.9998     0.23673   -1.2349     0.        
Gnucap   System status
iterations: op=5, dc=0, tran=0, fourier=0, total=50
transient timesteps: accepted=0, rejected=0, total=0
nodes: user=4, subckt=0, model=0, total=4
dctran density=75.0%, ac density=75.0%
